js/client.js

```diff
@@ -1636,7 +1651,19 @@
 			document.title = room.title ? room.title + " - Showdown!" : "Showdown!";
 		},
 		updateAutojoin: function () {
-			if (Config.server.id !== 'showdown') return;
+			if (!Config.server.registered) return;
+			var curAutojoin = (Tools.prefs('autojoin'));
+			var useString = false;
+			if (typeof curAutojoin === 'string' || curAutojoin === undefined) {
```

Comment:
This is pretty verbose code.

How many lines can you save with `var curAutojoin = (Tools.prefs('autojoin') || '');`?